prometheus_hardware_exporter/collectors/redfish.py

```diff
@@ -221,7 +221,29 @@ def _storage_root_uri(cls, system_id: str, storage_name: str) -> str:
           system_id: "S1"
           storage_name" "Storage"
         """
-        return cls.systems_root_uri + f"{system_id}/{storage_name}/"
+        return cls.systems_root_uri + f"{system_id}/{storage_name}"
+
+    def _find_storage_controller_data(self, storage_controllers: Dict) -> List[Dict]:
+        """Return storage controller list.
+
+        Returns:
+            storage_controller_list: list of storage controllers for 
+            requested storage id.
+
+        """
+        if "StorageControllers" in storage_controllers:
+            storage_controllers_list: List[Dict] = storage_controllers.get("StorageControllers")
+        elif "Controllers" in storage_controllers:
+            storage_controllers_list: List[Dict] = []
+            controller_list = storage_controllers.get("Controllers")
+            for controller in controller_list.values():
+                all_controller_data = self.redfish_obj.get(controller).dict
```

Comment:
docs: It would also be nice to have a comment here to understand what we're expecting the `controller_list` to look like, to better understand why we need another nested for loop querying for the `Members` key.

Eg: Maybe we could mention that `Controllers` only returns a list of links to the storage controllers when fetched from `Members`. And to get individual storage controller data, we need to query via separate GET requests to the URI pointed by `@odata.id`?